src/migrations/migrate.rs

```diff
@@ -263,8 +263,8 @@ async fn fixup(cli: &mut Connection, ctx: &Context,
         match db_migration.generated_by {
             Some(MigrationGeneratedBy::DevMode) => {
                 return Err(
-                    anyhow::anyhow!("Database contains Dev mode migrations.")
-                    .hint("Use `edgedb migrate --dev-mode` or `edgedb watch`")
+                    anyhow::anyhow!("Database contains Dev mode / `edgedb watch` migrations.")
+                    .hint("Use `edgedb migration create` followed by `edgedb migrate --dev-mode`, or resume `edgedb watch`")
```

Comment:
@Dhghomon In the case that you've decided to leave dev mode and create the actual migration, wouldn't your `edgedb migrate` exclude the `--dev-mode` flag? You'd only need that to continue in dev mode, I think…